mvn/README.md

```diff
@@ -1,10 +1,22 @@
 # Tool builder: `gcr.io/cloud-builders/mvn`
 
-This Cloud Build builder runs Maven. It also includes a number of dependencies
-that are precached within the image.
+This Cloud Build builder runs Maven.
+
+You should consider instead using an [official `maven`
+image](https://hub.docker.com/_/maven/) and specifying the `mvn` entrypoint:
```

Comment:
yuck, at least in code review, this is rendering wrong due to the '_' in the path.